### PR TITLE
Removed inconsistency in navigator

### LINKED ISSUE #6860: [UI] Inconsistency in the sidebar icons
#### Current Behavior

Only few icons in the sidebar  get blue, when user has selected them.
#### Desired Behavior

All icons in the sidebar shold get blue, when user has selected them. So that, user can know where he is.
#### Screenshots/Logs

https://user-images.githubusercontent.com/103871275/211491333-f2b90467-b52c-4c8e-9566-416d2b2acb07.mp4

#### Environment

- Browser: Firefox
- Host OS:Linux 
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Kubernetes

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by hs05june**
**Notes for Reviewers**

This PR fixes #6860 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

https://user-images.githubusercontent.com/103871275/211804769-fa984c31-c8d4-4f0f-8380-da2c93a89270.mp4

### CODE REVIEW

**Comment by leecalcote** [comment]
@hs05june commits from @Abhishek-kumar09 and @sudo-NithishKarthik are included in this PR by accident, I assume. Will you sync your fork/clone with `head` in order to ensure that only your commits are included?

File: ui/assets/icons/LifecycleIcons/TraekFileIcon.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+const TraekFileIcon = (props) => {
```
Comment (Harshit-Dandriyal) [inline ui/assets/icons/LifecycleIcons/TraekFileIcon.js:3]: Can you rename it, TraefikMeshIcon

**Review by Harshit-Dandriyal — CHANGES_REQUESTED** [review]
Everything looks cool except the traefikMesh naming. nice work..!

File: ui/assets/icons/LifecycleIcons/TraekFileIcon.js (hunk 1 above)
Comment (hs05june) [reply ui/assets/icons/LifecycleIcons/TraekFileIcon.js]: Done @Harshit-Dandriyal

**Review by Harshit-Dandriyal — APPROVED** [review]

**Review by Abhishek-kumar09 — COMMENTED** [review]
I think this is not the approach we would go for. I know that you've been seeing some other SVGs like this, but in the last dev call we discussed to remove those svgs as they were written.

Additonally, https://docs.meshery.io/project/contributing/contributing-ui#static-files-and-icons has the norm that svg files should be written with ".svg" extension and should follow certain conventions. We dont want react components for svg file unless very required. 

As you know, the colors, height width, and other things can be configured using [this approach](https://stackoverflow.com/a/62532186). Would you like to give it a try?

**Comment by hs05june** [comment]
@Abhishek-kumar09 I will be trying it that way

**Comment by hs05june** [comment]
@Abhishek-kumar09 I tried the approach suggested by you. But it could set only the attributes of the svg tag.
Code Image
To ensure that the icons have correct color, I have also changed the attributes of the path, polygon and g tag. It wasn't working properly.

**Comment by hs05june** [comment]
@Abhishek-kumar09 please suggest any other approach

**Comment by leecalcote** [comment]
Uh-oh. Some merge conflicts have cropped up.

**Comment by leecalcote** [comment]
@hs05june will you be attending to these conflicts?

**Comment by leecalcote** [comment]
Merge conflicts...

**Comment by hs05june** [comment]
@leecalcote  Yeah looking into them.

**Comment by leecalcote** [comment]
Circling back to review. Still merge conflicts.

**Comment by hs05june** [comment]
Hey @leecalcote . I had a word with @Abhishek-kumar09. He suggested me another better approach to solve this issue. I am closing this PR. I will be making another PR for this issue using what @Abhishek-kumar09 has suggested.

**Comment by leecalcote** [comment]
Ok. Understood.

**Pull request closed without merging**